eclipselink/src/test/groovy/data_manager/DataManagerPropertyConditionTest.groovy

```diff
@@ -190,4 +191,49 @@ class DataManagerPropertyConditionTest extends DataSpec {
 
         list.contains(testEntity3)
     }
+
+    def "load using PropertyCondition for collections"() {
+
+        TestAppEntity testAppEntity1 = dataManager.create(TestAppEntity)
+        testAppEntity1.name = 'test one'
+
+        TestAppEntity testAppEntity2 = dataManager.create(TestAppEntity)
+        testAppEntity2.name = 'test two'
+
+        TestAppEntityItem appEntityItem1 = dataManager.create(TestAppEntityItem)
+        appEntityItem1.name = 'one two'
+        appEntityItem1.appEntity = testAppEntity1
+
+        TestAppEntityItem appEntityItem2 = dataManager.create(TestAppEntityItem)
+        appEntityItem2.name = 'three'
+        appEntityItem2.appEntity = testAppEntity2
+
+        dataManager.save(testAppEntity1, testAppEntity2, appEntityItem1, appEntityItem2)
+
+        when:
+
+        def list = dataManager.load(TestAppEntity)
+                .condition(
+                        LogicalCondition.and()
+                                .add(PropertyCondition.contains("items.name", "one"))
+                                .add(PropertyCondition.contains("items.name", "two"))
+                )
+                .list()
+
+        then:
+
+        list == [testAppEntity1]
+
+        when:
+        list = dataManager.load(TestAppEntity)
+                .condition(
+                        LogicalCondition.and()
+                                .add(PropertyCondition.contains("items.appEntity.name", "two"))
```

Comment:
I guess that if the property is like `parent.items.name` (parent - is a many-to-one reference and then `items` is a collection of this parent), then it will not work. Did you expect such cases to work?